agenta-web/src/pages/apps/[app_id]/testsets/index.tsx

```diff
@@ -41,6 +42,8 @@ export default function Testsets() {
     const classes = useStyles()
     const router = useRouter()
     const appId = router.query.app_id as string
+    const [testsetIdsToDelete, setTestsetIds] = useState<Array<string>>([])
+    const [isConfirmDeleteModalOpen, setIsConfirmDeleteModalOpen] = useState(false)
```

Comment:
These states can be removed once we use `AlertPopup` in favor of `ConfirmTestsetDeleteModal`

Reply:
Ah, I see. Thank you.